README.md

```diff
@@ -161,17 +160,13 @@ SAFE = ""
 CHAIN_ID = ""
 DELEGATE_ADDRESS = ""
 DELEGATOR_ADDRESS = ""
-DELEGATOR_PRIVATE_KEY = ""
-
-ETH_NODE_URI_RINKEBY=""
-MNEMONIC_RINKEBY=""
+PRIVATE_KEY = ""
```

Comment:
This one need to be named `DELEGATOR_PRIVATE_KEY`, called in the .py script

Reply:
Actually you ned both, because if you ctrl+f PRIVATE_KEY, you need the private key of the delegate somewhere